docs/encoders.md

```diff
@@ -39,3 +43,112 @@ That's it, now on every reboot the motor will turn in one direction until it fin
 
 * If you wish to scan for the index pulse in the other direction (if for example your axis usually starts close to a hard-stop), you can set a negative value in `<axis>.encoder.config.idx_search_speed`.
 * If your motor has problems reaching the index location due to the mechanical load, you can increase `<axis>.motor.config.calibration_current`.
+
+### Startup sequence notes
+
+The following are variables that MUST be set up for your encoder configuration. Your values will vary depending on your encoder:
+
+* `<axis>.encoder.config.cpr = 4000`
+* `<axis>.encoder.config.mode = ENCODER_MODE_INCREMENTAL`
+
+The following are examples of values that MAY impact the success of calibration. Your values will vary depending on your setup:
+* `<axis>.motor.config.motor_type = MOTOR_TYPE_HIGH_CURRENT`
+* `<axis>.encoder.config.cpr = 4000`
+* `<axis>.encoder.config.calib_range = 0.05`
```

Comment:
I think these should be explained a little so people don't blindly change settings with no idea what they do lol

Reply:
I added: "The following are examples of values that MAY impact the success of calibration. Do not use these values unless you understand why they are needed; your values will vary depending on your setup:"

and gave some annotation for each one.